lib/mix/tasks/gettext/translate.ex

```diff
@@ -169,8 +107,11 @@ defmodule Mix.Tasks.Gettext.Translate do
   end
 
   # Gets the content of msgid of a `.pot` file.
+  # Replaces " with ' so that the output doesn't break.
   defp domain_line(line) do
-    Regex.scan(~r/(?<=\")(.*?)(?=\")/, line)
+    text = String.replace(line, ~r/\\\"/, "'")
```

Comment:
📓 This fixes the translation issue. In our first pass, we replace all instances of `\"` with `'`.